Resources/Prototypes/Objectives/thief.yml

```
@@ -129,10 +129,10 @@
   - type: StealCondition
     stealGroup: IDCard
     minCollectionSize: 5
-    maxCollectionSize: 15
+    maxCollectionSize: 10
     verifyMapExistence: false
   - type: Objective
-    difficulty: 0.7
+    difficulty: 1
```

```suggestion
    difficulty: 0.7
```
The number reduction is good, but keep the old difficulty, please.